app/src/main/java/xterminators/spellingbee/UserFunctions.java

```diff
@@ -18,9 +20,13 @@ public UserFunctions() {
     }
 
     /** The path to the dictionary file for the game. */
-    public static final String DICTIONARY_PATH = "src\\dictionary_optimized.txt";
+    public static final String DICTIONARY_PATH
+        = "app" + File.separator + "src" + File.separator + "main" + 
```

Comment:
Maybe we should put File.separator in a constant with a short name just to improve readability. Not a code problem, but having "File.separator" all over the place makes it harder to read.

Reply:
How would you feel about something like `Paths.get("app", "src", "main", "resources", "dictionary_optimized.txt").toString()`, which would return the same thing? That way we don't we don't need a short name variable that would add its own readability concerns.

Reply:
Yeah, that would probably be the best option. A string with a bunch of concatenation is always confusing to read.

Reply:
Added this in commit 68c185ddbc3f7e29f1048afb22967308f8bebd7d.